icechunk-python/python/icechunk/credentials.py

```diff
@@ -340,15 +340,13 @@ def azure_credentials(
     raise ValueError("Conflicting arguments to azure_credentials function")
 
 
-def containers_credentials(
-    m: Mapping[str, AnyS3Credential] = {}, **kwargs: AnyS3Credential
-) -> dict[str, Credentials.S3]:
+def containers_credentials(m: Mapping[str, AnyS3Credential]) -> dict[str, Credentials.S3]:
```

Comment:
Should this be AnyCredential instead of AnyS3Credential

Reply:
yes!

Reply:
I'll need to think a bit about what's the right way to fix this. Interestingly, it was already a bug before this PR. I guess a survivor from a time where we only supported S3 virtual chunks